runtime/src/validator_manager.rs

```diff
@@ -135,26 +132,10 @@ impl<T: Config> pallet_session::SessionManager<T::ValidatorId> for Pallet<T> {
 	fn start_session(_start_index: SessionIndex) {}
 }
 
-impl<T>
-	pallet_session::historical::SessionManager<
-		T::ValidatorId,
-		Exposure<T::AccountId, T::CurrencyBalance>,
-	> for Pallet<T>
-where
-	T: Config + pallet_staking::Config,
-	T: pallet_session::Config<ValidatorId = <T as frame_system::Config>::AccountId>,
-{
-	fn new_session(
-		new_index: SessionIndex,
-	) -> Option<Vec<(T::ValidatorId, Exposure<T::AccountId, T::CurrencyBalance>)>> {
-		<Self as pallet_session::SessionManager<_>>::new_session(new_index).map(|r| {
-			r.into_iter()
-				.map(|v| {
-					let exposure = ExposureOf::<T>::convert(v.clone()).unwrap_or_default();
-					(v, exposure)
-				})
-				.collect()
-		})
+impl<T: Config> pallet_session::historical::SessionManager<T::ValidatorId, ()> for Pallet<T> {
```

Comment:
Restored back to rococo version, I have had to edit it to make it friendly with staking.